src/java/fr/paris/lutece/portal/util/mvc/xpage/MVCApplication.java

```diff
@@ -91,11 +91,11 @@
     private static final String CONTENT_TYPE_JSON = "application/json";
     private static final String CONTENT_TYPE_XML = "application/xml";
     private static Logger _logger = MVCUtils.getLogger( );
-    private List<ErrorMessage> _listErrors = new ArrayList<>( );
-    private List<ErrorMessage> _listInfos = new ArrayList<>( );
-    private List<ErrorMessage> _listWarnings = new ArrayList<>( );
-    private MVCMessageBox _messageBox;
-    private Controller _controller = getClass( ).getAnnotation( Controller.class );
+    private transient List<ErrorMessage> _listErrors = new ArrayList<>( );
+    private transient List<ErrorMessage> _listInfos = new ArrayList<>( );
+    private transient List<ErrorMessage> _listWarnings = new ArrayList<>( );
+    private transient MVCMessageBox _messageBox;
+    private transient Controller _controller = getClass( ).getAnnotation( Controller.class );
```

Comment:
As before, this does not make the class properly serializable.

Reply:
will be rollbacked